c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -624,21 +622,24 @@ else
 	touch $@
 endif
 
-.build/node_modules.timestamp: package.json
+node_modules/.bin/npm:
+	npm install npm
+
+.build/node_modules.timestamp: package.json node_modules/.bin/npm
 	mkdir -p $(dir $@)
 ifeq ($(NGEO), TRUE)
-	npm install
+	node_modules/.bin/npm install
```

Comment:
Maybe put `node_modules/.bin/npm` in a variable? You are typing that quite often...